ui-v2/tests/acceptance/page-navigation.feature

```diff
@@ -47,7 +47,7 @@ Feature: page-navigation
     | kv        | kvs        | /dc-1/kv/0-key-value/edit                                | /v1/session/info/ee52203d-989f-4f7a-ab5a-2bef004164ca?dc=dc-1&ns=@namespace | /dc-1/kv            |
     # | acl       | acls       | /dc-1/acls/anonymous                                     | /v1/acl/info/anonymous?dc=dc-1                                             | /dc-1/acls         |
 # These Endpoints will be datacenters due to the datacenters checkbox selectors
-    | token     | tokens     | /dc-1/acls/tokens/ee52203d-989f-4f7a-ab5a-2bef004164ca   | /v1/catalog/datacenters                                                     | /dc-1/acls/tokens   |
+    # | token     | tokens     | /dc-1/acls/tokens/ee52203d-989f-4f7a-ab5a-2bef004164ca   | /v1/catalog/datacenters                                                     | /dc-1/acls/tokens   |
```

Comment:
Probably just delete this instead of commenting now you've added the nicer individual navigation test (I 'think' you can delete the one I must've commented out a long time ago below here on line 52 also? Thats now covered in the new test you write right?